### ISSUE TITLE
Error: Duplicate entry '198140-198142' for key 'unicity'

### ISSUE THREAD

**Issue by Megachip** [fusioninventory/fusioninventory-for-glpi#2666]
This night I discovered some errors in my sql-errors.log which seems not should happened.
How can I (help to) solve/resolve this?  
GLPI 9.2.4, FI 9.2+2.0 
```
*** MySQL query error:
  SQL: INSERT
                   INTO `glpi_networkports_networkports` (`networkports_id_1`,`networkports_id_2`) VALUES ('198140','198142')
  Error: Duplicate entry '198140-198142' for key 'unicity'
  Backtrace :
  inc/commondbtm.class.php:504                       
  inc/commondbtm.class.php:928                       CommonDBTM->addToDB()
  .../inc/inventorynetworkequipmentlib.class.php:456 CommonDBTM->add()
  .../inc/inventorynetworkequipmentlib.class.php:336 PluginFusioninventoryInventoryNetworkEquipmentLib->importConnectionMac()
  .../inc/inventorynetworkequipmentlib.class.php:152 PluginFusioninventoryInventoryNetworkEquipmentLib->importPorts()
  ...inc/communicationnetworkinventory.class.php:298 PluginFusioninventoryInventoryNetworkEquipmentLib->updateNetworkEquipment()
  ...inc/communicationnetworkinventory.class.php:530 PluginFusioninventoryCommunicationNetworkInventory->importDevice()
  ...inventory/inc/inventoryruleimport.class.php:795 PluginFusioninventoryCommunicationNetworkInventory->rulepassed()
  inc/rule.class.php:1436                            PluginFusioninventoryInventoryRuleImport->executeActions()
  inc/rulecollection.class.php:1461                  Rule->process()
  ...inc/communicationnetworkinventory.class.php:373 RuleCollection->processAllRules()
  ...inc/communicationnetworkinventory.class.php:206 PluginFusioninventoryCommunicationNetworkInventory->sendCriteria()
  ...inc/communicationnetworkinventory.class.php:123 PluginFusioninventoryCommunicationNetworkInventory->importContent()
  ...fusioninventory/inc/communication.class.php:235 PluginFusioninventoryCommunicationNetworkInventory->import()
  ...fusioninventory/inc/communication.class.php:463 PluginFusioninventoryCommunication->import()
  plugins/fusioninventory/front/communication.php:93 PluginFusioninventoryCommunication->handleOCSCommunication()
  plugins/fusioninventory/index.php:59               include_once()
```

**Comment by Stoatwblr** [comment]
It's still doing it even with latest bleeding edge 9.3 plugin and agents.

**Comment by ddurieux** [comment]
Fixed in 37b94a7964ab2a04e9c2e260d42aef5601ac2d75
